collectors/googlestackdriver/cfn/googlestackdriver-collector.template

```diff
@@ -57,6 +57,10 @@
             "Description": "JSON list of google resources to poll logs from. In the format <resourceType>/<resourceID>",
             "Type": "String"
         },
+        "LogNameFilters": {
```

Comment:
It should be with URI encoded , I am adding validation in application 
`{
          "property": "log_name_filter",
          "label": "Filter",
          "description": "List of Google Cloud log names used to filter logs. The format should be 'service.googleapis.com%2Flog_type', for example: 'cloudaudit.googleapis.com%2Factivity'.",
          "type": "string[]/textarea",
          "updateNotAllowed": true,
          "validationPattern": "([a-z]+\\.googleapis\\.com%2F[a-zA-Z0-9\\-]+[\\r\\n ]*)*"
        },`

Reply:
If the log names are already URI encoded -- e.g. if the setup would be "copy and paste already-URI-encoded log names from GCP into Applications UI" -- then it makes sense that the expected input would be URI encoded since that's the path of least resistance.  

If the log names aren't already URI encoded in GCP though, and the setup would be "copy and paste log names from GCP into Applications UI, then replace `/` with `%2F` manually to satisfy the validation pattern", then I think it makes more sense to expect the input to be not-URI-encoded, and then we can handle the encoding ourselves. 

I'm happy to keep it as-is for now so as to unblock testing of the collector itself, but let's keep this in mind as an ergonomics improvement

Reply:
Handle both encode and non encoded logname